### PR TITLE
more shared test coverage

### PR DESCRIPTION
**Pull request by kkoskelin**

### CODE REVIEW

File: shared/src/business/entities/cases/Case.js [hunk 1]
```diff
@@ -219,7 +219,7 @@ function Case(rawCase, { applicationContext, filtered = false }) {
         correspondence =>
           new Correspondence(correspondence, { applicationContext }),
       )
-      .sort((a, b) => compareStrings(a.createdAt, b.createdAt));
+      .sort((a, b) => compareStrings(a.filingDate, b.filingDate));
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.js:222]: Was this change to fix a bug?

File: shared/src/business/entities/cases/Case.test.js [hunk 2]
```diff
@@ -54,6 +54,41 @@ describe('Case entity', () => {
     });
   });
 
+  it('sorts correspondence array according to createdAt', () => {
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.test.js]: Should be according to `filingDate`?

File: shared/src/business/entities/cases/Case.test.js [hunk 3]
```diff
@@ -54,6 +54,41 @@ describe('Case entity', () => {
...
+  });
+  describe('conditionally sets userId on entity', () => {
+    it('sets userId to current user if current user matches rawCase', () => {
+      const myCase = new Case(
+        { ...MOCK_CASE, userId: applicationContext.getCurrentUser().userId },
+        { applicationContext },
+      );
+      expect(myCase.userId).toEqual(applicationContext.getCurrentUser().userId);
+    });
+    it('does NOT set userId to current user if current user matches rawCase', () => {
```
Comment (rachaelparris) [inline shared/src/business/entities/cases/Case.test.js]: `if current user does not match rawCase`?

File: shared/src/business/entities/cases/Case.js (hunk 1 above)
Comment (kkoskelin) [reply shared/src/business/entities/cases/Case.js:222]: Correct. Correspondence entities do not have a `createdAt` attribute.

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**